packages/core/src/utils/styleUtils/ssrStylesDesignTokenResolution.spec.tsx

```diff
@@ -180,18 +180,16 @@ describe('custom component with builtInStyles which are supporting design tokens
     expect(
       (customComponent.variables.cfSsrClassName as DesignValue).valuesByBreakpoint
         .desktop as string,
-    ).toEqual(
-      'cf-ae92b9eb2c20a262573a8085db3dbd67 cf-1fe22311d2b832dd529a9ce5e8d77281 cf-63009ce9ca644a23e001721d6358e491',
-    );
+    ).toEqual(expect.stringMatching(/cf-\w{32} cf-\w{32} cf-\w{32}/));
```

Comment:
is the point of this test - to validate that the hash matches the regexp or is it to make sure that after a change in the PR the hash stays the same compared to the main branch of the project?

This change makes the test check the hash format, which I think is not the intention

Reply:
Judging by the test name and previous code lines, I think this test was not about testing a specific hash string. To reduce the dependency on the outcome of `md5`, it only checks now that three class names are created but doesn't anymore fail when changing the hash generation function itself.

If we really want to test the result of `md5()`, I would do this in a separate test instead of all tests checking this.

Reply:
[Postponing the discussion and reverting my change]